package.json

```diff
@@ -19,7 +19,6 @@
     "react-modal": "^3.5.1",
     "react-spinners": "^0.3.2",
     "react-transition-group": "^2.4.0",
-    "redux-logger": "^3.0.6",
```

Comment:
What is the reason to remove redux logger?

Reply:
it was a merge issue, it's back in